assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -99,6 +104,21 @@ function TemplateDetail() {
     });
   }, [templatesOrderById, templates]);
 
+  const relatedTemplates = useMemo(() => {
```

Comment:
Even though we may not need a new Wordpress api endpoint to fetch related templates (because we should be able to use the external templates fetch), I still feel we should have a `getRelatedTemplates` method in our `useTemplateApi` file that'll do all this magic logic.  This way we keep as much of the temp code in our api, and our components work as if the api were working.

Reply:
And if you do migrate the logic to `useTemplateApi`, I don't think you need to worry about the `orderedTemplates`.  It doesn't matter what order our random templates are in anything like that, we just need a random set from the collection of available templates to show.

Reply:
Makes sense, thanks for suggesting it!

Reply:
(i'd mostly used the orderedTemplates because it was already mapped to be the structure the grid needed, but still this can all be done separated and smoother as part of the template context and passed through)